src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.VSCodeWorkspaces/WorkspacesHelper/VSCodeWorkspacesApi.cs

```diff
@@ -106,6 +110,50 @@ public List<VSCodeWorkspace> Workspaces
                                     }
                                 }
                             }
+                            else if (File.Exists(vscode_storage_db))
```

Comment:
Should this be inside `if (File.Exists(vscode_storage))`. Does db logic depend on existence of `vscode_storage` file? I.e. can we still read workspaces from DB if there is no `vscode_storage` file?

Reply:
No, we can't read workspaces if there's no vscode_storage.

Reply:
👍